seedlet/assets/sass/components/header/_config.scss

```
@@ -33,4 +33,7 @@
 	--social-nav--color-link: var(--global--color-foreground);
 	--social-nav--color-link-hover: var(--global--color-primary-hover);
 	--social-nav--padding: calc( 0.5 * var(--primary-nav--padding) );
+
+	--marketing-bar--height: 45px;
```

How about renaming this to be context specific? 

 ```suggestion
	--wpcom--marketing-bar--height: 45px;
```